README.md

```diff
@@ -223,6 +224,34 @@ The value has to have only numeric values.
   }
 ````
 
+### CustomValidation
+Define a custom callback funciton to validate the model's value. The validation callback is passed 3 values: the key, value, model's scope. return true (or a truthy value) to pass the validation, return false (or falsy value) to fail the validation.
+
+````js
+  validations: {
+    lotteryNumber: {
+      custom: funciton(key, value, model){
```

Comment:
passing the `key` and `value` make validating the value less verbose and reusable if many properties share common custom validation. the 3rd param is important to access other properties on the model, this way we can validate properties that have dependencies on other properties. Im having a hard time coming up with an example off the cuff, but for example: a user's password cannot be equal to their username.

```js
validations: {
  password: {
    custom: function (key, value, _this) { // var key = 'password', var value = 'MyPassword123', _this = (the model's 'this')
      var password = value;
      var username = _this.get('username');
      return (password !== username);
    }
  }
}
```